SDK_CHANGELOG.md

```diff
@@ -5,6 +5,16 @@
 
 ## Releases
 
+### v0.18.0 - Tech Preview (10/23/2018)
+
+* Following CloudBackup APIs have been refactored to include task id rather than
+* source volume id.
```

Comment:
There is a bullet in this one, but it is a continuation of the previous sentence.

Reply:
yep. will fix that.